### ISSUE TITLE
Magic eden floor price ticker returns `Bad data` error

### ISSUE THREAD

**Issue by rickstaa** [rssnyder/discord-stock-ticker#179]
**Describe the bug**

Thanks for the amazing tool! I just saw in the codebase that you also included support for collections found on Magic Eden:

https://github.com/rssnyder/discord-stock-ticker/blob/92c6092bfc64ae03516e2069a68bb2140ee647ed/utils/floor.go#L11-L16

Unfortunately, when I try to use this feature, I get the following error:

```bash
INFO[0000] Running discord-stock-ticker version development...
WARN[0000] Will not be storing state.
INFO[0000] Starting api server on 0.0.0.0:8080...
DEBU[0003] Got an API request to add a ticker
INFO[0003] Added ticker: floor
INFO[0004] Watching stock price for floor
DEBU[0014] Fetching stock price for lily
ERRO[0014] Yahoo returned bad data for lily
```

As I couldn't find this feature in the documentation,, I wondered if it is already released.

**To Reproduce**
Steps to reproduce the behaviour:
1. Clone [the repository](https://github.com/rssnyder/discord-stock-ticker) or run [the docker container](https://github.com/rssnyder/discord-stock-ticker/pkgs/container/discord-stock-ticker).
2. Save the following bot JSON file as `lily_fp.json` in the `bots` folder:

```json
{
  "ticker": "floor",
  "marketplace": "magiceden",
  "name": "lily",
  "nickname": true,
  "frequency": 10,
  "discord_bot_token": "<BOT_TOKEN>"
}
```

4. Create a bot, as explained in https://www.youtube.com/watch?v=LhgCdtE8kmc.
5. Add the bot token to the JSON file.
6. Start the discord-stock-ticker `./discord-stock-ticker -logLevel 1`.
7. Post the floor price bot `curl -X POST -H "Content-type: application/json" -d @bots/lily_fp.json localhost:8080/ticker`.
8. See error.

**Expected behaviour**
The bot should successfully update according to the collection floor price.

**Python (please complete the following information):**
 - Version: 3.8.10
 - OS: Ubuntu 20.04 and docker

**Comment by rickstaa** [comment]
I found the solution the correct place to put the ticker is in the curl command:

```bash
curl -X POST -H "Content-type: application/json" -d @bots/lily_fp.json localhost:8080/floor
```

What confused me was the `ticker` json object found in the [stock price configuration documentation](https://github.com/rssnyder/discord-stock-ticker#bot-configuration-stock).

**Comment by rickstaa** [comment]
Closed by #180.
